hack/create-subscription.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+oc create namespace kube-service-catalog || true
+
+cat <<'EOF' | oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: svcat
+  namespace: kube-service-catalog
+spec:
+  channel: alpha
+  name: svcat
+  source: rh-operators
+  installPlanApproval: Automatic
+  catalogSourceNamespace: kube-service-catalog
+---
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: service-catalog
+  namespace: kube-service-catalog
+spec:
+  selector:
+    matchLabels:
+      ns: kube-service-catalog
```

Comment:
How does this label match the namespace created above? I'd expect to see something like `oc label ns kube-service-catalog ns=kube-service-catalog`.

Reply:
I added a label to the kube-service-catalog namespace as suggested.  I can't tell you why it worked without it, but it did...  Anyway, I agree this looks more obvious now, and also validated it still works.